lib/testing/gpuBeamformSimulate.cpp

```diff
@@ -449,6 +449,9 @@ void gpuBeamformSimulate::main_thread() {
             }
         }
 
+        //16-bandpass correction
+        float BP[16] {0.52225748,0.58330915,0.6868705,0.80121821,0.89386546,0.95477358,0.98662733,0.99942558,0.99988676,0.98905127,0.95874124,0.90094667,0.81113021,0.6999944,0.59367968,0.52614263};
```

Comment:
This likely won't pass the clang-format line length rule (100 characters).   Although looks like the CI server isn't working for some reason...  I will look into that.